pkg/command/install/version/verison.go

```diff
@@ -14,6 +14,32 @@
  * limitations under the License.
  */
 
-package install
+package version
 
-type Task func(option interface{}) error
+import (
+	"fmt"
+
+	"golang.org/x/mod/semver"
+)
+
+const (
+	MinimumVersion = "v0.2.0"
+
+	LatestVersion = "latest"
+)
+
+func ValidateVersion(version string) (bool, error) {
+	if version == LatestVersion {
+		return true, nil
+	}
+
+	if !semver.IsValid(version) {
+		return false, fmt.Errorf("input version[%s] is not valid", version)
+	}
+
+	r := semver.Compare(version, MinimumVersion)
+	if r == -1 {
+		return false, fmt.Errorf("input version cannot be lower than %s", MinimumVersion)
+	}
```

Comment:
I guess it would be better to write like this:
```suggestion
	if semver.Compare(version, MinimumVersion) < 0 {
		return false, fmt.Errorf("version must be >= %s", MinimumVersion)
	}
```